src/home_robot/home_robot/manipulation/heuristic_pick_policy.py

```diff
@@ -0,0 +1,221 @@
+from typing import Any, Dict, Optional
+
+import cv2
+import numpy as np
+import torch
+
+import home_robot.utils.depth as du
+from home_robot.core.interfaces import (
+    ContinuousFullBodyAction,
+    ContinuousNavigationAction,
+    DiscreteNavigationAction,
+    Observations,
+)
+from home_robot.manipulation.heuristic_place_policy import HeuristicPlacePolicy
+from home_robot.utils.image import smooth_mask
+from home_robot.utils.rotation import get_angle_to_pos
+
+RETRACTED_ARM_APPROX_LENGTH = 0.15
+STRETCH_STANDOFF_DISTANCE = 0.6
+STRETCH_GRASP_DISTANCE = 0.4
+STRETCH_EXTENSION_OFFSET = 0.0
+ANGLE_ADJUSTMENT = -0.01
+
+
+class HeuristicPickPolicy(HeuristicPlacePolicy):
```

Comment:
We should add some documentation to this class

Also is it something I should be using on the real robot?

Reply:
This is just used to visualize the agent reaching the object in sim - so it's not necessary to test this as this is not part of our baseline either in sim/real.

Reply:
Added documentation